lib/get-seam.ts

```diff
@@ -1,25 +1,37 @@
-import { SeamHttp } from "@seamapi/http/connect"
+import { SeamHttp, SeamHttpMultiWorkspace } from "@seamapi/http/connect"
 import { getConfigStore } from "./get-config-store"
 import { getServer } from "./get-server"
 
-export const getSeam = () => {
+export const getSeam = async (): Promise<SeamHttp> => {
```

Comment:
This was being `await`ed in the code, so I made it async, but it could be sync as written.